ai.py

```
@@ -0,0 +1 @@
+trint("hello Code Rabbit!!")
```

Ruff (0.14.6)

Undefined name `trint`

(F821)